### PR TITLE
Fix several compiling issues for FreeBSD

### PR DESCRIPTION
**Pull request by lsun100**
Signed-off-by: Liming Sun <limings@nvidia.com>

### CODE REVIEW

File: src/rshim_pcie.c [hunk 1]
```diff
@@ -192,7 +191,7 @@ typedef struct {
 int bf3_rshim_pcie_chan_map[] = {
```
Comment (hselasky) [inline src/rshim_pcie.c:191]: BTW: Should this array be "static const int bf3_rshim_pcie_chan_map[]" ?

**Comment by hselasky** [comment]
Changes look good!

File: src/rshim_pcie.c (hunk 1 above)
Comment (lsun100) [reply src/rshim_pcie.c]: Done. Updated version 2.

**Review by hselasky — APPROVED** [review]
Looks good. Just make a new tag aswell and I'll get it into the FreeBSD ports tree!

**Comment by lsun100** [comment]
> Looks good. Just make a new tag as well and I'll get it into the FreeBSD ports tree!

Thanks. Done and tagged as rshim-2.0.6-10

**Comment by lsun100** [comment]
Thanks, done. I re-created rshim-2.0.6-10.

**Comment by hselasky** [comment]
Now upstream FreeBSD:
https://cgit.freebsd.org/ports/commit/?id=66be6b83209a10aa59cc5576ac9144f2c0a44f51

**Comment by lsun100** [comment]
Nice!

**Pull request merged**